pom.xml

```diff
@@ -3,12 +3,12 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>edu.gatech</groupId>
-    <artifactId>VRDR</artifactId>
-    <version>v1.4.0-STU2</version>

```

Comment:
Can you also edit the README.md release section at the bottom with a note to update the version number in the pom.xml?  I think we should start doing that with each release.

Reply:
README updated. The publishing workflow only runs on new releases and manually; I will double check the execution once the PR is merged.